grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -283,6 +286,101 @@ class StockMovementController {
         render(template: "documents", model: [stockMovement: stockMovement])
     }
 
+    def comments = {
+        def stockMovement = getStockMovement(params.id)
+        render(template: "comments", model: [stockMovement: stockMovement])
+    }
+
+    def addComment = {
+        StockMovement stockMovement = stockMovementService.getStockMovement(params.id)
```

Comment:
@drodzewicz now I started to think if this is a good idea performance-wise. If you should not use requisition here directly. For huge stock requests, this might end up being a bit slow (here and in every one of these actions)

Reply:
Kacper made a good point about moving these comment methods to RequisitionService since comments are only attached to the requisition, so I'll probably do that and resolve this issue too.

Reply:
the problem is that I still need `stockMovement` to render the summary header 
```gsp
<g:render template="summary" model="[stockMovement:stockMovement]" />
```
unless it is not necessary then I think I can replace all of the stockMovements I use with requisition